runbotics/runbotics-orchestrator-ui/src/views/process/ProcessBuildView/Modeler/ConfigureActionPanel/Actions/browser.actions.ts

```diff
@@ -681,6 +681,7 @@ const getBrowserActions: () => Record<string, IBpmnAction> = () => ({
         translateKey: 'Process.Details.Modeler.Actions.Browser.PrintToPdf.Label',
         script: 'browser.selenium.printToPdf',
         runner: Runner.DESKTOP_SCRIPT,
+        system: ActionSystem.WINDOWS,
```

Comment:
so this only works on windows for now? `getChromiumPath` is kinda tricky

Reply:
yup, if we can get chrome on linux we can refactor this